lib/index.coffee

```diff
@@ -3,4 +3,5 @@ module.exports =
   ProductImport: require './product-import'
   PriceImport: require './price-import'
   ProductDiscountImport: require './product-discount-import'
-  EnumValidator: require './enum-validator'
\ No newline at end of file
+  EnumValidator: require './enum-validator'
+  UnknownAttributesFilter: require './unknown-attributes-filter'
```

Comment:
dnt know... without this it gives me an error -> `{UnknownAttributesFilter} = require '../lib'` statement in the test file resolves to undefined.
No compile errors though.